src/test/java/org/reaktivity/reaktor/test/annotation/Configure.java

```diff
@@ -0,0 +1,31 @@
+/**
+ * Copyright 2016-2017 The Reaktivity Project
+ *
+ * The Reaktivity Project licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+package org.reaktivity.reaktor.test.annotation;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Repeatable;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+@Retention(RetentionPolicy.RUNTIME)
+@Target({ ElementType.METHOD })
+@Repeatable(Configures.class)
+public @interface Configure
```

Comment:
Rename to `ReaktorConfiguration` ?

Reply:
The configure matches the `ReaktorRule#configure`, how about ReaktorConfigure and ReaktorConfigures for the annotations?

Or, if we change it to ReaktorConfiguration, should we change the builder to be Configuration.

Reply:
`ReaktorConfigure` and `ReaktorConfigures` works!